apps/news/news.arc

```diff
@@ -570,8 +567,8 @@
 ; in case anyone else was confused, 'posts* and 'events*
 ; become bound if blog.arc and events.arc are included.
 
-    (if (bound 'posts*) (toplink "blog" "blog" label))
```

Comment:
Was this deliberate? (Just checking. I don't think this is particularly serious breakage, famous last words :)

Reply:
I can't actually get the blog to run... I don't know why, and I can't remember the error at the moment so I left it out.

Reply:
Cool. Is that independent of this change? If so, I'd recommend committing this hunk separately from the rest of this PR.

Reply:
fair enough.